composer/packages/theme/src/definitions/views/api-designer.less

```diff
@@ -29,22 +29,68 @@
             .action-container {
                 margin-bottom: 1em;
             }
-
-            .accordion.ui.fluid.styled {
-                padding-bottom: .5em !important;
+            .accordion {
                 margin-top: 20px;
                 border-left: 0;
                 border-right: 0;
                 border-top: 1px solid #e8e8e8;
                 border-bottom: 1px solid #e8e8e8;
```

Comment:
Move all color codes to `ballerina-lang\composer\packages\theme\src\themes\default\viewsapi-designer.variables` file and use.

Reply:
Will be tracked with issue #13334